### DISCUSSION TITLE
Infinity keeps on crashing on Android 13

### DISCUSSION THREAD

**Discussion by rohandebsarkar** [Docile-Alligator/Infinity-For-Reddit#1159] [Q&A]
I just installed an Android 13 ROM on my phone. All the apps are running normally except Infinity for Reddit. On opening the app, the animations are laggy and ultimately after about 10-15 second of usage it crashes. This happens every time I try to use the app. I tried force stopping the app, clearing all the data, and also uninstall and then reinstalling the app. But the app keeps on crashing.

Is there any way I could debug the app and know why it crashes? So that I can file a bug report.

**Comment by rohandebsarkar** [comment]
I just noticed the app crashes only when I try to play a video.

**Reply by Docile-Alligator** [reply]
Could you go to Settings->Advanced->Crash reports and provide the crash log?

**Reply by rohandebsarkar** [reply]
I created an issue  #1160.
